### DISCUSSION TITLE
add PWA capabilities

### DISCUSSION THREAD

**Discussion by 1nc1n3r470r** [dani-garcia/vaultwarden#3956] [Ideas]
i dont know whats missing to be honest, but a PWA would be nice. As far as i can see the manifest.json would need a "start_url" and "display" param

**Comment by BlackDex** [comment]
I'm not sure that would be a right thing to do.
There are already apps for all major platforms and devices. A PWA would not add much since it will not integrate into the system.

Also, Vaultwarden doesn't maintain the web-vault. We only make small patches to get it working for Vaultwarden and want to leave as much as possible the same as the original.

So, maybe you can try to post this at Bitwarden.

**Reply by stefan0xC** [reply]
There already is a feature request: https://community.bitwarden.com/t/make-bitwarden-a-progressive-web-app-pwa/10161/

**Comment by 1nc1n3r470r** [comment]
true, i didnt know how the front-end works so i made a post here xD
